book.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self unless book.rental == self
+  end
```

Comment:
- [ ] Just like the first reviewer has said, nice job implementing the add_rental method, however, the book itself should be sent as a parameter to create the new rental, thereby achieving the 'has-many' association. We can achieve this by modifying our code to look like the following example below:
```
class Book
  attr_accessor :title, :author, :rentals
  .
  .
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```